plugins/py3embed/py3embed.cpp

```diff
@@ -48,6 +48,10 @@
   #define Py_TPFLAGS_HAVE_ITER 0
 #endif
 
+#if PY_MINOR_VERSION < 7
```

Comment:
Should this be a check on major and minor ?
if PY_MAJOR == 3 && PY_MINOR < 7

Reply:
This plugin is only for python3, so not really needed

Reply:
ok, sure.  I was thinking this was the default python pluging going forward, I forgot it was specifically py3embed.